lib/tasks/sass.js

```diff
@@ -2,65 +2,66 @@
 
 const files = require('../helpers/files');
 
-function runSilentModeTest () {
+function runSilentModeTest() {
 	return silentCompilation()
 		.then(() => nonSilentCompilation());
 }
 
-const silentCompilation = function() {
+const silentCompilation = function () {
 	return silentCompilationTest(true);
 };
 
-const nonSilentCompilation = function() {
+const nonSilentCompilation = function () {
 	return silentCompilationTest(false);
 };
 
 function silentCompilationTest(silent) {
 	const sass = require('./build-sass');
 	const silentSass = require('../plugins/silent-sass');
 	const nodeSass = require('node-sass');
+	return files.getMainSassPath()
```

Comment:
This stuff here is quite difficult to read, Is it possible for the `then`s to be chained in a more flat way?